src/business-logic/services/rundown-timeline-service.ts

```diff
@@ -154,6 +156,11 @@ export class RundownTimelineService implements RundownService {
     this.rundownEventEmitter.emitSetNextEvent(rundown)
     this.startAutoNext(timeline, rundownId)
 
+
+    if (rundown.getActiveSegment().definesShowStyleVariant) {
```

Comment:
Missing tests

Reply:
A test has been added that verifies that the reingest is called if a segment has the new property set to true.

Reply:
You have an if-statement. That means atleast two tests. One for going into the if-statment and one for not going into the if-statement. 
I made a comment on your test.